workflows/diagnostics/fv3net/diagnostics/_shared/transform.py

```diff
@@ -278,10 +278,17 @@ def _mask_array(
 @add_to_input_transform_fns
 def subset_variables(variables: Sequence, arg: DiagArg) -> DiagArg:
     """Subset the variables, without failing if a variable doesn't exist"""
-    prognostic, verification, grid = arg.prediction, arg.verification, arg.grid
+    prognostic, verification, grid, delp = (
+        arg.prediction,
+        arg.verification,
+        arg.grid,
+        arg.delp,
```

Comment:
just to confirm—we don't use `arg.delp` in any of the online diags, right? I find it a bit confusing to have it separate from the prediction/verification data, since the pressure thicknesses might be different for those datasets.

Reply:
Correct